heltour/tournament/migrations/0021_auto_20160724_1855.py

```diff
@@ -20,7 +20,7 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='registration',
             name='already_in_slack_group',
-            field=models.BooleanField(),
```

Comment:
Altering old migrations has no effect on the live database, what was the intent with this change?

Reply:
the intent was simply to make the new migrations revertible. as far as i can tell, that's the only thing it does, and i didn't see a downside to it. i don't mind removing the changes to the old migrations from the pull request.

Reply:
I am pretty sure there are ways to make the new migrations invertible that don't include modifying old migrations. The problem with modifying old migrations is that our live database will not be in sync with this change, which means that new migrations may assume they can do things that they can't and it will succeed in development, but not on prod. Let's remove the changes to old migrations and if you want the new migrations to be revertible, we can work through that.